Material_TimePicker/src/main/ets/components/ChipTextInputComboComponent.ets

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (C) 2022 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TimePickerModel } from "./timepickermodel"
+import { ElementFormat, ClockPeriod, TimeFormat, InputMethod } from "./Types"
+
+@Component
+export struct ChipTextInputComboComponent {
+  @Link timePickerModel: TimePickerModel
+  @State selectedPeriod: number= 0
+  hourButtonClickTime: Date
+  minuteButtonClickTime: Date
+  StateColors: ResourceColor[]= ["#ffd8ddff", "#e4e4e4"]
+  TimeStateColors: ResourceColor[]= ["#ff5e00ff", "#ff282828"]
+  periodBGColors: ResourceColor[]= ["#ffd8ddff", Color.White]
+  periodStateColors: ResourceColor[]= ["#ff5e00ff", "#ffabaaad"]
+  borderColor: ResourceColor= "#ffabaaad"
+
+  aboutToAppear() {
+    if (this.timePickerModel.model.period == ClockPeriod.PM)
```

Comment:
Add even the single statement of if condition within braces

Reply:
done